unity/CAVE/Assets/Resources/CAVE.unity

```diff
@@ -20,11 +20,11 @@ RenderSettings:
   m_FogDensity: 0.01
   m_LinearFogStart: 0
   m_LinearFogEnd: 300
-  m_AmbientSkyColor: {r: 1, g: 1, b: 1, a: 0}
+  m_AmbientSkyColor: {r: 0.21176471, g: 0.22745098, b: 0.25882354, a: 0}
```

Comment:
alpha of zero doesnt make it complete transparent? does it change at any time ?

Reply:
It doesn't seem to? This is me changing it back to the default value when you create an empty scene. The main color of the scene comes from the background color of the camera